tests/suit/migration_test.go

```diff
@@ -36,12 +36,27 @@ var _ = Describe("[level:component]Migration tests for vSphere provider", func()
 		err = utils.WaitForProviderReadyWithTimeout(f.CrClient, f.Namespace.Name, providerName, 30*time.Second)
 		Expect(err).ToNot(HaveOccurred())
 		provider, err := utils.GetProvider(f.CrClient, providerName, f.Namespace.Name)
+
```

Comment:
did you mean to add an empty line after the next line?..

Reply:
fixed